composer.json

```diff
@@ -105,6 +105,7 @@
         "mockery/mockery": "^1.5.1",
         "orchestra/testbench-core": "^8.1",
         "pda/pheanstalk": "^4.0",
+        "pestphp/pest": "^2.0",
```

Comment:
This will prevent PHPUniit 9 to be installed if we need to debug any of our PHPUnit 9 integration.

Maybe add this on the build script instead? or allow v1 too

Reply:
Three lines below there is already phpunit 10 required so that would already be a problem.

Reply:
Our test suite only runs over PHPUnit 10 at the moment.

Reply:
So are we dropping PHPUnit 9 support for Laravel 10? because this would affect Testbench dependencies and any packages supporting Laravel 10 with an older version.

Our test suite doesn't run PHPUnit 9 because we want to reduce the number of builds, but we can still use PHPUnit 9. Of course, breaking changes on patch release is unlikely but If in the future there's any new deprecation issue introduced in PHPUnit 9.x we still need to run the test suite with PHPUnit 9 (unless we do intend to remove it).

Reply:
We are not planning on removing PHPUnit v9 support for Laravel v10. Right now, we don't want to run duplicate tests on PHPUnit v9 because we'd have to juggle configurations and CI settings too much.